pylib/anki/importing/anki2.py

```diff
@@ -356,13 +360,13 @@ def _importCards(self) -> None:
                 card[8] = card[14]
                 card[14] = 0
                 # queue
-                if card[6] == 1:  # type
-                    card[7] = 0
+                if card[6] == QUEUE_TYPE_LRN:  # type
```

Comment:
perhaps that one should be CARD_TYPE_LRN?

Reply:
Corrected